README.md

```diff
@@ -27,6 +27,20 @@ Note: `src/backend/sinr` is a submodule from `UMassCDS/inatrualist-sinr`, if you
 
 Note: If you need to switch to a particular `UMassCDS/inaturalist-sinr` branch and run the prototype, navigate to `src/backend/sinr`, use `git checkout <branch-name>` to switch to a branch. Now the submodule will be at a different branch.
 
+## Working with database
+
+1. Notice that there are two .env files, .docker.env and .env, .env is used for your local development, .docker.env will be used in your containers.
+
+2. To run the application without containers, first run the database container by `docker compose up db` (you may have to rebuild it if you changed docker-compose.yml). Run `uvicorn src.backend.app.main:app --reload --env-file .env` at project root to run the server, then `npm start` in `src/frontend`.
```

Comment:
There is a docker section in the readme, this section should go first or maybe be referenced here. Users might be blindsided when looking at the docker command

Reply:
Docker compose up db should be one step. 
Open a different terminal, run uvicorn should be another step. 
Open a third terminal, run npm start should be another step.

Reply:
To run uvicorn and npm, we first need to set up the Conda environment and run npm I, why are these steps after? they should be higher up!